pytest_hot_reloading/plugin.py

```diff
@@ -117,6 +120,16 @@ def apply_assertion_rewrite(self, ast_func, glb):
                         nodes.append(field)
             return ast_func
 
+        def stash(self, lineno=1, col_offset=0):
+            if not isinstance(self.parent, OrigFunctionDefinition):
+                co = self.get_object()
+                if co and (delta := lineno - co.co_firstlineno):
+                    delta -= 1  # fix off-by-one
```

Comment:
This is the fix for the off-by-one bug. I'm guessing this logic exists for a reason, so its likely not the best fix. However, seeing what breaks (if anything) when we add this may help illuminate why it is here in the first place